package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hyperproof/hypersync-sdk",
-  "version": "0.10.0",
+  "version": "1.0.0",
```

Comment:
Insertion of `orgId` parameter to some HypersyncApp methods constitutes a breaking change.